src/main/java/org/apache/commons/crypto/stream/CryptoInputStream.java

```diff
@@ -587,10 +587,10 @@ static void freeDirectBuffer(final ByteBuffer buffer) {
             .clean(); */
             final String SUN_CLASS = "sun.nio.ch.DirectBuffer";
             final Class<?>[] interfaces = buffer.getClass().getInterfaces();
+            final Object[] NO_PARAM = new Object[0];
```

Comment:
You can use the same name we have in Commons Lang: EMPTY_OBJECT_ARRAY.

Reply:
Ok, changed